cmd/update-main.go

```diff
@@ -62,15 +63,22 @@ EXAMPLES:
 `,
 }
 
-const releaseTagTimeLayout = "2006-01-02T15-04-05Z"
-
-const minioReleaseURL = "https://dl.minio.io/server/minio/release/" + runtime.GOOS + "-" + runtime.GOARCH + "/"
+const (
+	minioReleaseTagFormat = "2006-01-02T15-04-05Z"
```

Comment:
I think the const name needs to be properly named to differentiate release tag format and time portion in release tag.